### PR TITLE
Add next button to the book cover.

### PR DESCRIPTION
**Pull request by etefera**
And removes previous button from last page.
Fixes https://github.com/GoogleContainerTools/kpt/issues/1720
image

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -98,12 +98,33 @@
...
         paginationFooters.forEach(
           (el) => (el.style.display = isBookPage ? "flex" : "none")
         );
+
+        // Don't show previous button on the book cover page.
+        const previousPaginationButtons = Array.from(
+          document.getElementsByClassName("pagination-item--previous")
+        );
+        const isBookCover =
+          isBookPage && document.location.pathname.toLowerCase().length < 7;
```
Comment (mikebz) [inline site/index.html:113]: length is a little magical here.  Is there a way to just firmly say that the book cover is path `x`?

File: site/index.html [hunk 2]
```diff
@@ -98,12 +98,33 @@
...
+        previousPaginationButtons.forEach(
+          (el) => (el.style.display = isBookCover ? "none" : "flex")
+        );
+
+        // Don't show next button to non-book-pages.
+        const nextPaginationButtons = Array.from(
```
Comment (mikebz) [inline site/index.html:119]: is the strategy here to add the next/previous to everything and then hyde it via a style?  Is there a way to just have book pages derive from some template that has those and the rest of the pages don't have that?  It looks like the javascript in index.html is getting a bit much.

File: site/index.html (hunk 1 above)
Comment (etefera) [reply site/index.html:113]: Added more precise matching.

File: site/index.html (hunk 2 above)
Comment (etefera) [reply site/index.html:119]: Yeah, the next/previous is already added to the site by a plugin. This is additional styling to control where it appears based on the URL. It seems that's the preferred method of plugin management from Docsify. I agree the inline JS is getting a bit cluttered; we should move what we can to a separate JS file (which we can also automatically test) at a later time.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**